UI.AutomationTests/Admin/Booking/EditMultiDayHearingTests.cs

```diff
@@ -0,0 +1,194 @@
+using UI.PageModels.Pages.Admin.Booking;
+
+namespace UI.AutomationTests.Admin.Booking
+{
+    public class EditMultiDayHearingTests : AdminWebUiTest
+    {
+        [Category("Daily")]
+        [Test]
+        public void EditSingleDayOfMultiDayHearing()
+        {
+            var isV2 = FeatureToggles.UseV2Api();
+            const int numberOfDays = 3;
+            var hearingDto = HearingTestData.CreateMultiDayDtoWithEndpoints(numberOfDays, DateTime.Today.AddDays(1).AddHours(10).AddMinutes(00));
+            var bookingDetailsPage = BookMultiDayHearingAndGoToDetailsPage(hearingDto);
+            UpdateCaseName(hearingDto, numberOfDays);
+            
+            // Change the scheduled datetime
+            var newTime = hearingDto.ScheduledDateTime.AddMinutes(30);
+            var summaryPage = bookingDetailsPage.UpdateSchedule(newTime, hearingDto.DurationHour, hearingDto.DurationMinute);
+            hearingDto.ScheduledDateTime = newTime;
+            
+            // Change everything else
+            summaryPage = EditMultiDayHearing(isV2, hearingDto, summaryPage);
+            
+            summaryPage.ValidateSummaryPage(hearingDto);
+            var confirmationPage = summaryPage.ClickBookButton();
+            
+            confirmationPage.IsBookingSuccessful().Should().BeTrue();
+            var bookingDetailPage = confirmationPage.ClickViewBookingLink();
+            bookingDetailPage.ValidateDetailsPage(hearingDto);
+            Assert.Pass();
+        }
+
+        [Category("Daily")]
+        [Test]
+        public async Task EditThisAndUpcomingDaysOfMultiDayHearing()
+        {
+            var isV2 = FeatureToggles.UseV2Api();
+            var multiDayBookingEnhancementsEnabled = FeatureToggles.MultiDayBookingEnhancementsEnabled();
+            if (!isV2 || !multiDayBookingEnhancementsEnabled)
+                Assert.Pass("V2 and multi day booking enhancements are not both enabled, cannot edit this and upcoming hearings. Skipping Test");
```

Comment:
Using `Assert.Ignore` will record the test as ignored rather than a pass
```suggestion
                Assert.Ignore("V2 and multi day booking enhancements are not both enabled, cannot edit this and upcoming hearings. Skipping Test");
```

or you can use the `Assume.That` method to specify preconditions for a test. If the assumption fails, the test will be ignored

``` suggestion
Assume.That(!isV2 || !multiDayBookingEnhancementsEnabled, "V2 and multi day booking enhancements are not both enabled, cannot edit this and upcoming hearings. Skipping Test");
```

Reply:
this does seem to be the case

with `Assert.Pass`, test skips and passes: https://app.eu-central-1.saucelabs.com/tests/37646cdba4704df9baf060764c6014da
with `Assert.Ignore`, test skips and fails: https://app.eu-central-1.saucelabs.com/tests/f0f299b0bf334e3a84af38ed5169a117

Reply:
If you modify TearDown in the AdminWebUITest and VideoWebUITest you can change the existing code from

```VhDriver.PublishTestResult(TestContext.CurrentContext.Result.Outcome.Status == TestStatus.Passed);```

to

```
bool passed = TestContext.CurrentContext.Result.Outcome.Status == TestStatus.Skipped ||
   TestContext.CurrentContext.Result.Outcome.Status == TestStatus.Passed;
VhDriver.PublishTestResult(passed);
```

Reply:
thanks. Updated